QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,145 @@
+This document is a declaration of software quality for the `foonathan_memory` package imported by `foonathan_memory_vendor`, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `foonathan_memory` Quality Declaration
+
+This quality declaration claims that the third party package `foonathan_memory` is in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`foonathan_memory` does not have a declared versioning scheme.
+
+### Version Stability [1.ii]
+
+`foonathan_memory` is in the beta stage, and currently has a version less than 1.0. The public API may change between minor version updates.
```

Comment:
Where does the "beta" declaration come from upstream? I guess I'm asking whether there's a source to cite for this or if it's based on implicit information and what information if so?

That the version number is below 1.0 communicates some meaning but nothing specific since there is no declared version scheme.

Reply:
beta is probably not the right word, since this package is pretty much in maintenance mode. I adjusted the language to reflect that API changes may occur during small version updates. Also added a cite to the changelog.